src/main/java/edu/kit/hci/soli/controller/LoginControllerAdvice.java

```diff
@@ -50,6 +52,6 @@ public LoginStateModel getLoginStateModel(Principal principal, @AuthenticationPr
             user.setUsername(username);
             userService.create(user);
         }
-        return new LoginStateModel(username, visitsRepository.getVisits(), LoginStateModel.Kind.OAUTH, csrfToken);
+        return new LoginStateModel(username, LoginStateModel.Kind.OAUTH, csrfToken, user);
```

Comment:
We need the `User` to add a booking to the database, so inject it with the state model